azure/cloud-config.yml

```diff
@@ -0,0 +1,47 @@
+azs:
+- name: z1
+  cloud_properties:
+    availability_zone: ((az))
+- name: z2
+  cloud_properties:
+    availability_zone: ((az))
+- name: z3
+  cloud_properties:
+    availability_zone: ((az))
+
+vm_types:
+- name: default
+  cloud_properties:
+    instance_type: Standard_D1
+- name: large
+  cloud_properties:
+    instance_type: Standard_D3
+
+disk_types:
+- name: default
+  disk_size: 3000
+- name: large
+  disk_size: 50_000
+
+networks:
+- name: default
+  type: manual
+  subnets:
+  - range: ((internal_cidr))
+    gateway: ((internal_gw))
+    azs: [z1, z2, z3]
+    dns: [168.63.129.16, ((secondary_dns))]
```

Comment:
i think we can just use 168... as default. if someone want to update it they can always write their own ops file.